docs/source/options/mp4_output_options.rst

```diff
@@ -17,3 +17,8 @@ MP4 output options
     template).
 
     Default enabled.
+
+--mp4_initial_sequence_number
```

Comment:
Thanks for working on this feature. Here is my high level feedback.

There are two concepts in this CL.

1. mp4_initial_sequence_number which specifies the initial sequence number in moof->mfhd.

2. the starting segment index.

They are different concepts. They may not have the same value as a segment can contain multiple fragments.


Can we have a separate flag for segment index, e.g. start_segment_number to match the `startNumber` in DASH `SegmentTemplate`?

I think we can remove the initial sequence number from this PR (or move it to a separate PR).

 I also think that it can be a ChunkingParam as it is not restricted to mp4.

Reply:
Thanks for the feedback.
`They are different concepts. They may not have the same value as a segment can contain multiple fragments.`
You are right. My mistake. 
I do not see exoplayer's FragmentedMp4Extractor.java parsing it 
https://github.com/google/ExoPlayer/blob/03263db378392385e290c40505d80e990c85a5cb/library/extractor/src/main/java/com/google/android/exoplayer2/extractor/mp4/FragmentedMp4Extractor.java#L708
What is the use-case for specifying moof->mfhd sequence number? 

```
Can we have a separate flag for segment index, e.g. start_segment_number 
to match the startNumber in DASH SegmentTemplate?
I also think that it can be a ChunkingParam as it is not restricted to mp4.
```
Yes that makes sense. Will look into it.

Reply:
> What is the use-case for specifying moof->mfhd sequence number?

I don't really know. None of the players and devices I know care about the sequence number in mfhd. That is actually the question I have in the original PR (#270) too.

Reply:
Yeah that makes sense.. we can keep it out. I don't know of use-cases either where it would be needed to change the initial sequence number of moof -> mfhd. 
From what I understand was that segment index start should be enough to fix console platform issues and align us with the DASH spec, that should be the only validation happening player side. Beyond that it just uses either index ranges or the indexed fragmented files from origin for delivery.